autoload.php

```diff
@@ -1,7 +1,6 @@
 <?php
-
 $autoloadFile = './vendor/codeception/codeception/autoload.php';
-if (file_exists('./vendor/autoload.php') && file_exists($autoloadFile) && __FILE__ != realpath($autoloadFile)) {
+if ( ( isset($argv) && isset($argv['--no-redirect']) ) && file_exists('./vendor/autoload.php') && file_exists($autoloadFile) && __FILE__ != realpath($autoloadFile)) {
```

Comment:
I think that you got this condition backwards, it should be `!isset($argv['--no-redirect'])`.

Reply:
Oops, sorry. You are right